lib/sufia/engine.rb

```diff
@@ -53,6 +53,10 @@ class Engine < ::Rails::Engine
       end
 
       CurationConcerns::CurationConcern.actor_factory = Sufia::ActorFactory
+      # Don't try to load this class until the application has been generated
+      if defined? ::SearchBuilder
+        CurationConcerns::AdminSetService.default_search_builder = Sufia::AdminSetSearchBuilder
+      end
```

Comment:
It would be nice to give a warning in an else.  Could make someone's upgrade path easier
```
else
  logger.warn(' You must run "rails g sufia:install" for the Admin Set Search to run correctly')
end
```